servers/rendering/renderer_rd/shaders/forward_clustered/SCsub

```diff
@@ -4,7 +4,7 @@ Import("env")
 
 if "RD_GLSL" in env["BUILDERS"]:
     # find all include files
-    gl_include_files = [str(f) for f in Glob("*_inc.glsl")]
+    gl_include_files = [str(f) for f in Glob("*_inc.glsl")] + [str(f) for f in Glob("../*_inc.glsl")]
```

Comment:
@clayjohn this should ensure that shaders are re-compiled when includes in parent folder change.

Should we put this in a separate PR or just wait until this is merged?

Reply:
This is now in https://github.com/godotengine/godot/pull/66423